### PR TITLE
MTP-2182 Animate Header dropdown

### PR DESCRIPTION
**Pull request by petterkoding**

### CODE REVIEW

File: mt-kit/core/css/src/components/_dropdown.scss [hunk 1]
```diff
@@ -421,7 +426,24 @@
     left: 0;
     right: 0;
     bottom: 0;
-    background: rgba(0, 0, 0, 0.15);
     z-index: var(--z-index-dropdown);
+    background: rgba(0,0,0,0);
+    pointer-events: none;
+
+    @media (prefers-reduced-motion: no-preference) {
+      transition: background 0.7s cubic-bezier(.38,.02,.26,1.01);
```
Comment (lha047) [inline mt-kit/core/css/src/components/_dropdown.scss:434]: Create variable for the cubic-function and the duration

File: mt-kit/core/svelte/src/lib/svelte/components/Dropdown.svelte [hunk 2]
```diff
@@ -55,15 +55,17 @@
...
-      <div in:slide={{ duration: 300 }}>
-        <slot {isOpen} />
+    {#if isOpen}
+      <div
+        class="dropdown-content"
+        id={bodyId}
+        use:clickOutside={titleId}
+        on:click={handleClick}
+        on:clickOutside={() => isOpen && send('TOGGLE')}>
+        <div in:slide={{ duration: 2200 }}>
```
Comment (lha047) [inline mt-kit/core/svelte/src/lib/svelte/components/Dropdown.svelte:65]: 2200 is a bit long?

Comment (petterkoding) [reply mt-kit/core/svelte/src/lib/svelte/components/Dropdown.svelte]: changed it to 900 and added out:slide for dismount transition

File: mt-kit/core/css/src/components/_dropdown.scss [hunk 3]
```diff
@@ -154,9 +153,6 @@
     & .mt-details > .mt-summary.mt-button {
       color: var(--color-mt-text-dark);
       .dropdown-content {
```
Comment (lha047) [inline mt-kit/core/css/src/components/_dropdown.scss:155]: Do you need this when there is nothing inside?

File: mt-kit/core/css/src/components/_dropdown.scss [hunk 4]
```diff
@@ -188,8 +181,9 @@
     }
 
     .dropdown-content {
-      display: inline-block;
-      min-width: min(100vw, 20rem);
+      > * {
```
Comment (lha047) [inline mt-kit/core/css/src/components/_dropdown.scss:184]: Dont think you need this as this should be on the content of the slot

**Review by lha047 — CHANGES_REQUESTED** [review]
Nice :D 

See comments

**Review by lha047 — APPROVED** [review]
Good job :D

**Pull request merged**